Scripts/CI/README_Metadata_StyleCheck/description_differ.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import json
+import plistlib
+import argparse
+from typing import List
+
+
+# region Global sets
+# A set of category folder names in current sample viewer.
+categories = {
```

Comment:
It looks like the list of categories exists in multiple places. Would it be possible to extract them into a separate file and then pass them in as an argument of the script? Just a thought.

Reply:
Yes. Eventually I want to combine all the helper functions and classes into a single file so that they are easier to maintain. Meanwhile I'll first put these checks into production. If they work well during this release, we can proceed with further optimization.